plugins/woocommerce-beta-tester/api/tracks/class-tracks-debug-log.php

```diff
@@ -44,16 +58,23 @@ public function __construct() {
 	 * @param array  $properties Event properties.
 	 */
 	public function log_event( $event_name, $properties ) {
-		$this->logger->debug(
-			$event_name,
-			array( 'source' => $this->source )
-		);
-		foreach ( $properties as $key => $property ) {
-			$this->logger->debug(
-				"  - {$key}: {$property}",
+		$logger = $this->get_logger();
+		
+		if ( ! $logger ) {
+			$logger->debug(
```

Comment:
Would this throw an error if a logger doesn't exist?

Reply:
Oops this should be `if ( $logger` lol thanks for spotting